### PR TITLE
Update TCK paths and README

### PR DESCRIPTION
**Pull request by mbroz2**
Fix https://openliberty.io/certifications/microprofile/4.0/TCKResults.html pointing to 20.0.0.3 instead of 21.0.0.3

Various updates/clarifications/cleanup in README.md

Change Jakarta EE TCKs path structure from `/certifications/jakartaee/<spec>/<version>/*` to `/certifications/jakartaee/<version>/<subspec>/*` and add redirects.

Add runtime version to all TCK file names that didn't have it and add redirects.

Delete previously redirected MP CCR TCKs.

----------------------------------------------------------

The above means that all TCKs now conform to the following URL/path structure (as defined in the README.md):
## Directory Structure
The URL structure on [openliberty.io/certifications/](https://openliberty.io/certifications/) directly matches the directory structure of this repo.
The following URL/path structure is to be used for this repo:

```https://openliberty.io/certifications/<spec>/<specVersion>/<optional:subSpec>/<optional:subSpecVersion>/<runtimeVersion>-TCKResults```

For example:
> [certifications/jakartaee/8/webprofile/19.0.0.6-TCKResults.adoc](https://openliberty.io/certifications/jakartaee/8/webprofile/19.0.0.6-TCKResults.html)  
> [certifications/jakartaee/9.1/platform/21.0.0.2-beta-Java11-TCKResults.adoc](https://openliberty.io/certifications/jakartaee/9.1/platform/21.0.0.2-beta-Java11-TCKResults.html)  
>  
> [certifications/microprofile/5.0/21.0.0.12-beta-TCKResults.adoc](https://openliberty.io/certifications/microprofile/5.0/21.0.0.12-beta-TCKResults.html)  
> [certifications/microprofile/5.0/config/3.0/21.0.0.12-beta-TCKResults.adoc](https://openliberty.io/certifications/microprofile/5.0/config/3.0/21.0.0.12-beta-TCKResults.html)  
>  
> [certifications/microprofile/contextpropagation/ccr/1.2/21.0.0.3-beta-TCKResults.adoc](https://openliberty.io/certifications/microprofile/contextpropagation/ccr/1.2/21.0.0.3-beta-TCKResults.html)

### CODE REVIEW

**Review by steven1046 — DISMISSED** [review]

**Review by brideck — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+```https://openliberty.io/certifications/<spec>/<specVersion>/<optional:subSpec>/<optional:subSpecVersion>/<runtimeVersion>-TCKResults```
+
+For example:
+> [certifications/jakartaee/8/webprofile/19.0.0.6-TCKResults.adoc](https://openliberty.io/certifications/jakartaee/8/webprofile/19.0.0.6-TCKResults.html)  
+> [certifications/jakartaee/9.1/platform/21.0.0.2-beta-Java11-TCKResults.adoc](https://openliberty.io/certifications/jakartaee/9.1/platform/21.0.0.2-beta-Java11-TCKResults.html)  
+>  
+> [certifications/microprofile/5.0/21.0.0.12-beta-TCKResults.adoc](https://openliberty.io/certifications/microprofile/5.0/21.0.0.12-beta-TCKResults.html)  
+> [certifications/microprofile/5.0/config/3.0/21.0.0.12-beta-TCKResults.adoc](https://openliberty.io/certifications/microprofile/5.0/config/3.0/21.0.0.12-beta-TCKResults.html)  
+>  
+> [certifications/microprofile/contextpropagation/ccr/1.2/21.0.0.3-beta-TCKResults.adoc](https://openliberty.io/certifications/microprofile/contextpropagation/ccr/1.2/21.0.0.3-beta-TCKResults.html)
```
Comment (tevans78) [inline README.md]: is the `ccr` part still needed?

Comment (mbroz2) [reply README.md]: @Emily-Jiang ^

Comment (Emily-Jiang) [reply README.md]: don't need ccr any more

**Review by Emily-Jiang — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (mbroz2) [reply README.md]: ok, then I can update all the paths to get rid of ccr, along with the one that was just delivered for for MP CP 1.3.

**Review by steven1046 — APPROVED** [review]

**Pull request merged**